jax/experimental/sparse/transform.py

```diff
@@ -70,26 +70,31 @@
 
 sparse_rules : Dict[core.Primitive, Callable] = {}
 
+# Zero-preserving unary primitives that commute with addition
```

Comment:
Is "linear unary primitives" a correct characterization? Simpler if so.

Reply:
Good call - Done!